egs2/voxceleb/spk1/run.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+set -e
+set -u
+set -o pipefail
+
+
+spk_config=conf/train_RawNet3.yaml
+
+train_set="voxceleb2_dev"
+valid_set="voxceleb1_test"
+test_sets="voxceleb1_test"
```

Comment:
Why is the test set used as the validation set?

Reply:
I'm sharing for others which we discussed offline.
VoxCeleb currently does not officially have a validation set.
Therefore, I wanted the recipe to show EER on the test set every epoch.